config.toml.in

```diff
@@ -1,46 +1,52 @@
-# An example of an optional config file for pick.
+# Pick configuration
+#
 # If no config file is available, sensible defaults will be used.
 # If you wish to use a config file, it must be located at ~/.pick/config.toml
+#
 # Variables:
-# $HOME gets replaced with the path to your home directory, e.g. $HOME/.pick -> /home/john/.pick
+# $HOME - Replaced with the path to your home directory, e.g. $HOME/.pick -> /home/john/.pick
 
-# The storage section controls the configuration of the backend.
+
+## Storage
+# The storage section controls the where the safe file is stored.
 [storage]
 type = "file"
   [storage.settings]
   path = "$HOME/.pick/pick.safe"
 
-# Danger Area!
-# Only adjust the settings below this line if you know what you are doing.
-# If you change something and we update our recommended settings, your safe will
-# _not_ automatically upgrade to our new recommendations.
-# The encryption section controls the configuration of the safe encryption.
-#[encryption]
-# type let's you specify the encryption method to use.
-# Currently supported values: "openpgp", "aes_gcm".
-#type = "openpgp"
-#type = "aes_gcm"
-  # Settings for the "openpgp" type
-  #[encryption.openpgp]
-  # The cipher to use, currently supported: "aes256", "aes128".
-  # The number after "aes" specifies the key length (in bits).
-  #cipher = "aes256"
-  # s2kcount sets the number of iterations used to derive a key from your password.
-  # The more iterations, the longer it takes to convert your password to a key.
-  # This increases the time required for e.g. dictionary-attacks.
-  # Currently supported values: 1024-65011712 (inclusive).
-  # Not all values in the 1024-65011712 range are legal and if an illegal value is selected,
-  # the value is automatically rounded up to the nearest legal value.
-  #s2kcount = 65011712
-  # Settings for the "aes_gcm" type
-  #[encryption.aes_gcm]
-  # Specifies the key length (in bytes) to use for AES.
-  # Currently supported values: 32, 24, 16 for 256bit, 192bit, 128bit respectively.
-  #keylen = 32
-  # The hash function to use for PBKDF2.
-  # Currently supported values: "sha512", "sha256".
-  #pbkdf2hash = "sha512"
-  # The number of iterations used to derive the key from your password.
-  #pbkdf2iterations = 100000
-  # Specifies the salt length (in bytes) for the PBKDF2.
-  #pbkdf2saltlen = 16
+
+## Encryption
+#
+#    Danger Area!
+#    Only adjust the settings below this line if you know what you are doing.
+#    If you change something and we update our recommended settings, your safe will
+#    _not_ automatically upgrade to our new recommendations.
+#    The encryption section controls the configuration of the safe encryption.
+#    type let's you specify the encryption method to use.
```

Comment:
`type = "aes_gcm"` has now moved to the bottom. We should make it clear, that only `type = "openpgp"` or `type = "aes_gcm"` can be active.

I'd do something like this:

```toml
# [encryption]
#    type let's you specify the encryption method to use.
#    Either use "openpgp" or "aes_gcm".
# type = "openpgp"
#
#   [encryption.openpgp]
# ...
#
#   [encryption.aes_gcm]
# ...
```